plugins/colorbutton/plugin.js

```diff
@@ -545,7 +545,7 @@
 						' data-value="' + this.color + '"' +
 						' role="option">' +
 						'<span class="cke_colorbox" style="background-color:#' + this.color + '"></span>' +
-					'</a>' );
+					'</a>', 'Created using this.color which is a value that is validated using a pattern and is not susceptible to DOM XSS attacks. This pattern is defined in CKEDITOR.tools._isValidColorFormat.') );
```

Comment:
And `this.label` is also sanitized in a similar way?

Reply:
We just talked about this, but because this is inside an attribute and not inside well-formed tags, we should also disallow quotes? Was it `.htmlEncodeAttribute`?

Reply:
Yep changed to [htmlEncodeAttr](https://sourcegraph.com/github.com/ckeditor/ckeditor4/-/blob/core/tools.js?L484-486)